### DISCUSSION TITLE
Run on windows?

### DISCUSSION THREAD

**Discussion by Shlok-Bhakta** [pine64/OpenPineBuds#67] [Q&A] [answered]
Would it be possible to run this on windows as is and maybe change some code so that the computer can emulate these earbuds. The reason is that I am an iPhone user *shame* and I cannot find an open source app that allows me to accept calls on my desktop and potentially even see the caller ID. Not sure how possible this is, but if someone has already done something similar, lmk what it is because I cannot seem to find it (intel unison broke on me a couple months ago).

**Comment by shymega** [comment, marked as answer]
Really don't understand your use case here. It's not possible to emulate the earbuds, it's a special chip.

Can you please elaborate on the iPhone part? The buds work with anything that supports Bluetooth Audio, and iPhones shouldn't be a factor.

Personally, I'm not sure you have the right project here...

**Reply by shymega** [reply]
Gonna mark this as the answer for now. Ping me if you feel my answer isn't correct.
